src/docker_snapshot/container.py

```diff
@@ -148,7 +151,11 @@ def sync(source_directory, destination_path):
         [
             "sh",
             "-c",
-            f"rsync -aAHX --delete --info=progress2 {source_directory}/ {destination_path}",
+            # TODO: this could probably be split on every whitespace character
+            (
+                "rsync -aAHX --delete --info=progress2 "
+                f"{source_directory}/ {destination_path}"
+            ),
```

Comment:
Note: had to interpolate this string (or the other option would've been to ignore this line), but I'm pretty sure every component of the string could be split on whitespace. Felt like keeping everything semantically identical for now though.